src/fmu/sumo/uploader/_sumofile.py

```diff
@@ -349,5 +349,26 @@ def upload_to_sumo(self, sumo_parent_id, sumo_connection):
             self._delete_metadata(sumo_connection, self.sumo_object_id)
         else:
             result["status"] = "ok"
+            file_path = self.path
+            metadatafile_path = _path_to_yaml_path(file_path)
+            if sumo_mode == "MOVE":
+                if os.path.exists(file_path):
+                    os.remove(file_path)
```

Comment:
Cannot put my finger on it, but this "feels" like something that could fail. Are there any "known" exceptions we should catch here? E.g. leave the file on disk if we (for some reason) cannot delete it, but not crash?

Reply:
Since RGS use network disks, we have even seen network errors on file operations, so hard to specify a small number of more precise exceptions here.

Reply:
Not sure what the best approach is here. We might want to tolerate failure to delete files over stopping the whole run. So I am wondering if this should go into a try/catch all (which would be bad practice, strictly speaking, but perhaps needed...). Perhaps others can voice in on this as well.

Reply:
Done